app/views/alchemy/pages/show.rb

```diff
@@ -131,6 +132,22 @@ def google_analytics_code
         ENV.fetch('GOOGLE_ANALYTICS_CODE', 'UA-123456-1')
       end
 
+      def as_json
```

Comment:
Move this into a Jbuilder view at `app/views/alchemy/pages/show.json.jbuilder`

Reply:
I had tried that originally, but a jbuilder template doesn't have access to page_object which mustache loads in via app/views/alchemy/pages/show.rb inheriting from Alchemy::Base.